openassessment/templates/openassessmentblock/staff_area/oa_staff_area.html

```diff
@@ -167,6 +167,7 @@ <h2 class="staff-info__title ui-staff__title">
         </div>
     </div>
 
+    {% if staff_assessment_required %}
```

Comment:
Note to reviewers: this section is hidden already if staff grade is not required. However, I thought it best in that case to also not render the HTML (which should never be un-hidden).